.circleci/config.yml

```diff
@@ -421,6 +421,67 @@ jobs:
       - coverage:
           flags: 'test_<< parameters.pg_major >>,upgrade'
 
+  test-query-generator:
```

Comment:
When I check CI, this one shows as optional. Is that intentional? I'd rather make it required.

Reply:
just to be clear, if we make this required, we can only make it required after we merge this PR that adds the CI job. Otherwise all other PRs are blocked.